client/src/main/resources/css/light.css

```
@@ -1,20 +1,89 @@
-.button{
-	-fx-background-color: orangered;
+* {
+	-fx-text-fill: black;
 }
 
-.list-cell:selected, .list-cell:selected > * {
-	-fx-background-color: orangered;
-	-fx-text-fill: black;
+.root {
+	-fx-background-color: #E6E6E6;
+}
+
+.button {
+	-fx-background-color: #B37D7D;
+}
+
+.button:pressed {
+	-fx-background-color: #808080;
+}
+
+.button:disabled {
+	-fx-background-color: lightgray;
 }
 
-.received-message, .menu-item {
+#message-list, .text-field, .password-field, .tooltip, .pane, .pane .content, .vbox, .titled-pane > .title, .titled-pane > *.content, .context-menu, .menu-item {
+	-fx-background-color: #E3E3E3;
+}
+
+.list-cell:selected, .list-cell:selected > *, .menu-item:hover {
+    -fx-background-color: #805959;
+}
+
+.received-message {
 	-fx-background-color: lightgray;
 }
 
 .own-message {
-	-fx-background-color: lightgreen;
+	-fx-background-color: #8fa88f;
```

I like it if we use only one method of hexadecimal: either all CAPS or all lowercase. Given that almost every other element is in CAPS (at least in this file), I'd suggest:
```suggestion
	-fx-background-color: #8FA88F;
```